Formula/zrok.rb

```diff
@@ -0,0 +1,36 @@
+require "language/node"
+
+class Zrok < Formula
+  desc "Geo-scale, next-generation sharing platform built on top of OpenZiti"
+  homepage "https://zrok.io"
+  url "https://github.com/openziti/zrok.git",
+    tag: "v0.3.4", revision: "03a3243694b9d3fc7af16b22c47b0de61f89b1dc"
+  license "Apache-2.0"
+  head "https://github.com/openziti/zrok.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GO111MODULE"] = "auto"
+    dir = buildpath/"src/github.com/openziti/zrok"
+    dir.install (buildpath/"").children
```

Comment:
for go module build, you dont need it.
```suggestion
```

Reply:
Applied suggested change, removing these buildpath declarations